Mathlib/SetTheory/Cardinal/Basic.lean

```diff
@@ -446,7 +446,7 @@ theorem le_one_iff_subsingleton {α : Type u} : #α ≤ 1 ↔ Subsingleton α :=
 theorem mk_le_one_iff_set_subsingleton {s : Set α} : #s ≤ 1 ↔ s.Subsingleton :=
   le_one_iff_subsingleton.trans s.subsingleton_coe
 
-alias ⟨_, _root_.Set.Subsingleton.cardinal_mk_le_one⟩ := mk_le_one_iff_set_subsingleton
+alias ⟨_, _root_.Set.Subsingleton.cardinalMk_le_one⟩ := mk_le_one_iff_set_subsingleton
```

Comment:
deprecated alias missing for this one

Reply:
cc @adomani, this one was missed by the script

Reply:
Ah, I had not thought of adding deprecations for *these* aliases!